common/src/main/scala/org/genivi/sota/db/SlickExtensions.scala

```diff
@@ -44,19 +44,23 @@ trait SlickExtensions {
       action.sortBy(fn).paginate(offset, limit)
 
     def defaultPaginateAndSort[T <% slick.lifted.Ordered]
-    (fn: E => T, offset: Option[Long], limit: Option[Long]): Query[E, U, Seq] =
+        (fn: E => T, offset: Option[Long], limit: Option[Long]): Query[E, U, Seq] =
       action.sortBy(fn).defaultPaginate(offset, limit)
 
-    def paginatedResult[T <% slick.lifted.Ordered](fn: E => T, offset: Option[Long], limit: Option[Long])
+    def paginatedResult[T <% slick.lifted.Ordered](offset: Option[Long], limit: Option[Long])
```

Comment:
This method is not using `T` so we can remove `T`, and then we will not get an ambiguity further down.